it/repositories/ApplicationRepositorySpec.scala

```diff
@@ -115,8 +116,11 @@ class ApplicationRepositorySpec extends MongoRepositorySpec {
         appStatus <- applicationRepo.findStatus(app.applicationId)
       } yield appStatus).futureValue
 
+      val hasCorrectStatusDate = Seconds.secondsBetween(DateTime.now(), applicationStatus.statusDate.get)
+        .isLessThan(Seconds.ONE)
```

Comment:
I'd perhaps consider boosting it to 5 to dodge any potential build breakage, this is my last comment on this PR